pkg/antctl/command_definition.go

```diff
@@ -88,6 +88,14 @@ type argOption struct {
 	key       bool
 }
 
+// flagInfo describes all needed information of a flag.
+type flagInfo struct {
+	name      string
+	shorthand string
+	value     string
```

Comment:
I know the cobra documentation uses "value", but I'd prefer if we called this "defaultValue" to be more explicit

Reply:
Yes, defaultValue is more clear. Updated.